docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Organize contracts with workspace
+hide_table_of_contents: true
+description: Organize your contracts using Cargo workspaces
+---
+
+### Initializing workspace project
+
+Using Cargo's workspace [feature](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) makes it very convenient to organize your smart contracts in subdirectories of your project's root.
+
+It's very simple to get started using the cli:
+
+```bash
+mkdir soroban-project
+cd soroban-project
+PROJECT_ROOT=.
+stellar contract init . --name add_contract
```

Comment:
This code here, and further down seem to be specifically targeting linux/mac, or bash at least. We need these tutorials where possible to get as portable as possible to windows. In some cases we might not be able to do that, but here we should be able to.

The use of the `PROJECT_ROOT=.` is also teaching more than we need to, it's teaching conventions that aren't required for using the tools and we should keep the tutorials as narrowly focused on the required actions as possible.

Suggest changing to:

````suggestion
```
stellar contract init soroban-project --name add_contract
````

For example, the above command will result in the following output and created directory structure:
```console
$ stellar contract init soroban-project --name add_contract
ℹ️ Initializing workspace at "soroban-project"
➕ Writing "soroban-project/.gitignore"
➕ Writing "soroban-project/Cargo.toml"
➕ Writing "soroban-project/README.md"
ℹ️ Initializing contract at "soroban-project/contracts/add_contract"
➕ Writing "soroban-project/contracts/add_contract/Cargo.toml"
➕ Writing "soroban-project/contracts/add_contract/Makefile"
➕ Writing "soroban-project/contracts/add_contract/src/lib.rs"
➕ Writing "soroban-project/contracts/add_contract/src/test.rs"
```